projects/vdk-core/src/vdk/internal/builtin_plugins/connection/managed_cursor.py

```diff
@@ -121,18 +123,28 @@ def execute(
                     )
 
         self._log.info("Executing query:\n%s" % managed_operation.get_operation())
+        query_start_time = timer()
         try:
             super().execute(*managed_operation.get_operation_parameters_tuple())
             self._log.info("Executing query SUCCEEDED.")
+            self._log.info(
+                f"Successful query {self._get_query_duration(query_start_time)}"
+            )
         except Exception as e:
             try:
                 self._recover_operation(e, managed_operation)
+                self._log.info(
+                    f"Recovered query {self._get_query_duration(query_start_time)}"
```

Comment:
Can you copy paste the log. How would it look like? 

Let's make it debug.

Reply:
'Recovered query duration H:M:S 0:01:02.103319'

> Let's make it debug.
Why?

Reply:
We have too many unnecessary logs. 

Let's see more user friendly formatting.

Reply:
New format is 'Recovered query duration 00h:00m:09s'

Reply:
We talked offline that for now we can leave them info and we will revisit the full default logging verbosity in a separate effort.